lib/order.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/menu.rb'
+require './lib/confirm_order.rb'
+
+class Order
+  attr_reader :order_list, :total
+  
+  def initialize
+    @menu = Menu.new
+    @order_list = []
+    @total = 0
+  end
+  
+  def print_menu
+    puts @menu.print_menu
+  end
+
+  def select_dish
+    puts "Please type in the dish you want:"
```

Comment:
It might be nice to be able to ignore case when responding here, you could add some more logic in to help with that perhaps?